vagrant/devmode/salt/roots/virtualenv.sls

```diff
@@ -29,6 +29,7 @@ pip_pkgs:
     - activate: true
     - requirements: /home/vagrant/calamari/requirements/2.7/requirements.txt
     - download_cache: /vagrant/pip_cache
+    - env_vars: SWIG_FEATURES=-cpperraswarn
```

Comment:
I mean, in the short term I could refactor m2crypto out of the requirements.txt file, and add a second call to pip in the salt file. Whatever you guys prefer.